recipes/wise_enum/all/conanfile.py

```diff
@@ -0,0 +1,48 @@
+from conans import ConanFile, tools
+from conans.errors import ConanInvalidConfiguration
+import os
+
+required_conan_version = ">=1.33.0"
+
+class WiseEnumConan(ConanFile):
+    name = "wise_enum"
+    description = "Header-only C++11/14/17 library provides static reflection for enums, work with any enum type without any boilerplate code."
+    topics = (
+        "cplusplus",
+        "enum-to-string",
+        "string-to-enum"
+        "serialization",
+        "reflection",
+        "header-only",
+        "compile-time"
+    )
+    homepage = "https://github.com/quicknir/wise_enum"
+    url = "https://github.com/conan-io/conan-center-index/"
```

Comment:
It looks like other recipes don't have a trailing `/`.
```suggestion
    url = "https://github.com/conan-io/conan-center-index"
```